translations/nl.po

```diff
@@ -236,10 +231,16 @@ msgstr "Stuur een email naar @emailAddress"
 msgid "View all contact info"
 msgstr "Bekijk alle contactgegevens"
 
-#: themes/contrib/gent_base/templates/contrib/paragraphs/timeline/paragraph--timeline-item--alt.html.twig:45
+#: themes/contrib/gent_base/templates/contrib/paragraphs/timeline/paragraph--timeline-item--alt.html.twig:83
 msgid "More info"
 msgstr "Meer informatie"
 
+#: themes/contrib/gent_base/templates/contrib/webform/webform--multistep.html.twig:39
+#: themes/contrib/gent_base/templates/contrib/webform/webform.html.twig:34
+#: themes/contrib/gent_base/templates/core/form/form.html.twig:15
+msgid "Fill in all fields unless otherwise stated."
+msgstr "Alle velden zijn verplicht, tenzij ze gemarkeerd zijn als optioneel."
```

Comment:
gemarkeerd => aangeduid?
Maar dit is wrs al ergens algemeen beslist..

Reply:
Inderdaad, ik heb deze niet vertaald, deze is enkel van plaats veranderd in de .po file.